Tests/JsonApiSerializerTests.cs

```diff
@@ -162,6 +162,27 @@ public void AppliesSorting()
             Assert.Equal(expected, props);
         }
 
+        [Fact(DisplayName = "Applies including when includes specified")]
+        public void AppliesIncluding()
```

Comment:
I think there needs to be a test that verifies that the `included` hash in the JSON API response has only one version of each resource, even if multiple resources refer to it.

Example:

1. Two people both have a job at the same company
2. Request all people and ask for the job to be included
3. Verify that the `included` hash in the JSON API document only has the company once.